runtime/smithy-client/common/src/aws/smithy/kotlin/runtime/client/config/SigV4aClientConfig.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package aws.smithy.kotlin.runtime.client.config
```

Comment:
**fix**: This doesn't belong in `smithy-client`. If anything it belongs in `aws-credentials` (which probably should have been named `aws-identity-api` or similar 🤷) or possibly `http-auth-aws`.